### DISCUSSION TITLE
<title>Updates to the actual site using GitHub pages

### DISCUSSION THREAD

**Discussion by jordanyan93** [community/community#57272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone. I am very new to programming and have been trying to build a personal website for myself to test my skills in React. I used GitHub pages to deploy my site. An issue that I have right now is that when I commit and push the changes I made in VS code, it shows in my repository (jordanyan93.github.io) that my files changed which is good. However, the actual site itself jordanyan93.github.io was not updated and hasn't been updated for a while.

Any help on this will be greatly appreciated.

**Comment by SRayen** [comment]
Hello, 
-Verify the branch being deployed: GitHub Pages can deploy from different branches.
-Wait for the changes to propagate: After pushing your changes to the repository, it may take some time for the changes to propagate and be reflected on the live website.
-Clear your browser cache or try another browser: Sometimes, your browser may cache the old version of the website, preventing you from seeing the latest changes. 
- Ensure that the files you modified or added have the correct paths and names. Pay attention to case sensitivity.
